data/gegno/gegno intro missions.txt

```diff
@@ -483,6 +800,22 @@ mission "Gegno Genocide Defense"
 			label "care"
 			`	The Quarg makes an unpleasant sound - one you have not heard from any Quarg before. "Should you be seen in Gegno space once more, expect the same sentiment."`
 				accept
+
+			label "corroboration provocation"
+			`As soon as you land your ship, you suddenly receive an incoming hail. "It was incredibly unwise to fire upon guardians of the Gegno homeworld." The voice belongs to that of a Quarg, though it sounds quite displeased. "They simply were present to quell unease of your foreign involvement, yet you chose hostility above their most sacred of planets. You shall find yourself no longer welcome in Gegno space, as well as at the end of our patience."`
+			choice
+				`	"I was worried they would attack me with their fleets!"`
+				`	"I do not care."`
+					goto care
+			`	"Your concerns matter not when weighed against those of an entire species. As a result of such behavior, the Gegno have banded together to unite their fleets and deter any more presence from outsiders. It is a great shame. The time they would use to resolve their inner conflicts is now wasted upon hostility towards strangers due to your naivety. Leave Gegno space, for you no longer have reason to be here."`
```

Comment:
```suggestion
			`	"Your concerns matter not when weighed against those of an entire species. As a result of such behavior, the Gegno have banded together to unite their fleets and deter any more presence from outsiders. It is a great shame. The time they would use to resolve their inner conflicts is now wasted upon hostility toward strangers due to your naïveté. Leave Gegno space, for you no longer have reason to be here."`
```

I'd prefer this spelling if we can use those characters - looks like this form and `naivete` are a bit more common in American English. But this is up to you; looks like any of the spellings is probably fine in this case. Just wanted to bring it to your attention.

Reply:
I think to avoid spelling and symbol confusion/problems, I'll change the word to "negligence" (which seems just as suitable).